<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PairTrading</class>
 <widget class="QWidget" name="PairTrading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PairTrading</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabMain">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QFrame" name="frame_2">
          <property name="minimumSize">
           <size>
            <width>421</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>421</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>82</height>
              </size>
             </property>
             <property name="title">
              <string>Add Pair</string>
             </property>
             <widget class="QPushButton" name="pushButtonRemoveSelected">
              <property name="geometry">
               <rect>
                <x>244</x>
                <y>50</y>
                <width>131</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Remove Selected</string>
              </property>
             </widget>
             <widget class="QWidget" name="layoutWidget">
              <property name="geometry">
               <rect>
                <x>20</x>
                <y>20</y>
                <width>355</width>
                <height>27</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="lineEdit"/>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit_2"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButtonAddPair">
                 <property name="text">
                  <string>Add</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QTableView" name="tableView">
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>200</height>
               </size>
              </property>
             </widget>
             <widget class="QGroupBox" name="groupBox">
              <property name="minimumSize">
               <size>
                <width>395</width>
                <height>245</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>395</width>
                <height>245</height>
               </size>
              </property>
              <property name="title">
               <string>GroupBox</string>
              </property>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QFrame" name="frame">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>631</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QTabWidget" name="tabWidget_2">
             <property name="currentIndex">
              <number>1</number>
             </property>
             <widget class="QWidget" name="tab_2">
              <attribute name="title">
               <string>Tab 1</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout_Tab"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tab_3">
              <attribute name="title">
               <string>Tab 2</string>
              </attribute>
              <widget class="QPushButton" name="pushButtonBuyMkt">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>70</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>Buy</string>
               </property>
              </widget>
              <widget class="QLineEdit" name="lineEditOrderSymbol">
               <property name="geometry">
                <rect>
                 <x>20</x>
                 <y>30</y>
                 <width>71</width>
                 <height>20</height>
                </rect>
               </property>
               <property name="text">
                <string>SPY</string>
               </property>
              </widget>
              <widget class="QSpinBox" name="spinBoxQuantity">
               <property name="geometry">
                <rect>
                 <x>110</x>
                 <y>30</y>
                 <width>71</width>
                 <height>22</height>
                </rect>
               </property>
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
              <widget class="QRadioButton" name="radioButtonBuy">
               <property name="geometry">
                <rect>
                 <x>200</x>
                 <y>20</y>
                 <width>82</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Buy</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
              <widget class="QRadioButton" name="radioButtonSell">
               <property name="geometry">
                <rect>
                 <x>200</x>
                 <y>40</y>
                 <width>82</width>
                 <height>17</height>
                </rect>
               </property>
               <property name="text">
                <string>Sell</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">buttonGroup</string>
               </attribute>
              </widget>
              <widget class="QPushButton" name="pushButtonSellMkt">
               <property name="geometry">
                <rect>
                 <x>100</x>
                 <y>70</y>
                 <width>75</width>
                 <height>23</height>
                </rect>
               </property>
               <property name="text">
                <string>Sell</string>
               </property>
              </widget>
             </widget>
            </widget>
           </item>
           <item>
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QGroupBox" name="groupBoxPrice">
              <property name="title">
               <string>Price</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3"/>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Page</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <property name="sizeConstraint">
          <enum>QLayout::SetDefaultConstraint</enum>
         </property>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
